sops.go

```
@@ -131,6 +184,111 @@ func (c *cipher) Encrypt(ctx context.Context, content []byte, config EncryptionC
+	}
+
+	updatedContent, err := store.EmitEncryptedFile(tree)
+	if err != nil {
+		return nil, fmt.Errorf("failed to emit encrypted file: %w", err)
+	}
+	return updatedContent, nil
+}
+
+func buildKeyGroups(ctx context.Context, config EncryptionConfig) ([]sops.KeyGroup, error) {
+	if len(config.Keys) > 0 && len(config.KeyGroups) > 0 {
+		return nil, errors.New("Keys and KeyGroups cannot be used together")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)